DevExtreme.AspNet.TagHelpers.Generator/IntellisenseData/Root.cs

```diff
@@ -0,0 +1,23 @@
+﻿using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Linq;
+using System.Threading.Tasks;
+using System.Xml.Serialization;
+
+namespace DevExtreme.AspNet.TagHelpers.Generator.IntellisenseData {
+
+
+    [XmlRoot("ArrayOfIntellisenseObjectInfo")]
+    public class Root {
```

Comment:
You mean removing the `Root` class?
This class is necessary, as we have "ArrayOfIntellisenseObjectInfo" as a root XML element. 

If we keep the XML file structure, we have to use this unnecessary wrapper (I mean the `Root` class).